fixtures/beamz/panther-7r.json

```diff
@@ -754,20 +660,31 @@
         {
           "dmxRange": [26, 100],
           "type": "Maintenance",
-          "comment": "Lamp off (Wait 3 seconds)"
+          "comment": "Lamp off",
+          "hold": "3s",
+          "helpWanted": "The manual specifies a hold time of 5s, which is correct?"
         },
         {
           "dmxRange": [101, 255],
           "type": "Maintenance",
-          "comment": "Lamp on (Wait 3 seconds)"
+          "comment": "Lamp on",
+          "hold": "3s",
+          "helpWanted": "The manual specifies no hold time, which is correct?"
```

Comment:
Paper manual specified 3 seconds of hold

Reply:
Okay, thanks for the information! Could you please verify this with the fixture itself to see which manual is correct? I'd keep the help wanted message for now. You can send a feedback message on the fixture page as soon as you have tested this.

Reply:
Yeah sorry, i already tested it and it's 3 seconds

Reply:
Great! Could you please also have a look at the other help wanted messages? It's already deployed here, so you can easily see them: https://open-fixture-library.org/beamz/panther-7r